Update manual steps in release process (#281)
PrestaShop/open-source: content/maintainers-guide/processes/release/prestashop/preliminary-tasks.md

  ```bash
  php bin/console prestashop:licenses:update
  ```
  
* All controllers are secured by annotations, and legacy link are provided for Symfony routes:
  
  ```bash
  php bin/console prestashop:linter:security-annotation
  php bin/console prestashop:linter:legacy-link

>   ```

Reviewer: This part mus be kept, but mentioning it is only necessary until 8.2

Reviewer: php bin/console prestashop:linter:security-annotation
  php bin/console prestashop:linter:legacy-link

  ```bash
  php bin/console prestashop:licenses:update
  ```

> * All controllers are secured by annotations, and legacy link are provided for Symfony routes (starying from 9.0.x this step is obsolete as it is checked in the CI already):

Reviewer 2: ```suggestion
* All controllers are secured by annotations, and legacy link are provided for Symfony routes (starting from 9.0.x this step is obsolete as it is checked in the CI already):
```

  ```bash
  php bin/console prestashop:licenses:update
  ```

> * Before PrestaShop 9.x: all controllers are secured by annotations, and legacy link are provided for Symfony routes:

Reviewer: ```suggestion
* All controllers are secured by annotations, and legacy link are provided for Symfony routes (starying from 9.0.x this step is obsolete as it is checked in the CI already):
```

  ```bash
  php bin/console prestashop:linter:security-annotation
  php bin/console prestashop:linter:legacy-link

> 
> * Before PrestaShop 9.x: There are no known vulnerabilities in composer dependencies using [Fabpot Local PHP Security Checker][security-checker]. Consider using [this][security-checker-installer] if installing Fabpot Security Checker proves troublesome.
> 
> * For PrestaShop 9.x and above: please execute `composer audit` to check vulnerabilities.

Reviewer: ```suggestion

* There are no known vulnerabilities in composer dependencies using `composer audit` to check vulnerabilities.
```

  php bin/console prestashop:licenses:update
  ```
  

> * Please execute `composer audit` to check vulnerabilities.

Reviewer: Ok for the replacement for security
However, we should let the steps for version 8.* and indicate that they are no longer need for versions 9+
